### ISSUE TITLE
ScopeCommand to get ommands from the region

### ISSUE THREAD

**Issue by ThePix** [textadventures/quest#911]
Sometimes it is useful to put a bunch of rooms inside a room that acts as a region. If you do that, it can be useful to have a command that applies to the region, rather than the entire game or a single room.

ScopeCommands would then be:

```
result = NewObjectList()
foreach (command, AllCommands()) {
  if (command.parent = null or command.parent = game.pov.parent) {
    list add (result, command)
  }
  if (not game.pov.parent.parent = null) {
    if (command.parent = game.pov.parent.parent) {
      list add (result, command)
    }
  }
}
return (result)

**Comment by ThePix** [comment]
See also here:
http://textadventures.co.uk/forum/quest/topic/zhzl_m8hd0qoookndtqrqq/inheriting-commands-from-the-direct-parent-object-is-it-possible

**Comment by ThePix** [comment]
Fixed for Quest 5.7.2
